utils_scraping.py

```diff
@@ -204,8 +204,9 @@ def web_files(*urls, dir=os.getcwd(), check=CHECK_NEWER):
 
 
 def sanitize_filename(filename):
-    return filename.translate({"*": "_", "?": "_", ":": "_", "\\": "_"})
-
+    return filename.translate(str.maketrans({"*": "_", "?": "_", ":": "_", "\\": "_", 
+                                                "<": "_", ">": "_","|": "_"}))
```

Comment:
Newly added.

Reply:
Yep, tested this change. I can successful complete a full run, when running locally on Windows.